blankiety_galantow/routers.py

```diff
@@ -1,10 +1,18 @@
 import codecs
+from random import randint
 from fastapi import APIRouter
 from fastapi.responses import HTMLResponse
+from blankiety_galantow.models.room import Room
+
 
 router = APIRouter()
 
 
 @router.get("/", response_class=HTMLResponse)
 async def root():
     return codecs.open("resources/index.html", "r", "utf-8").read()
+
+@router.get("/api/rooms")
+async def rooms():
+    rooms_dict = {i: vars(Room("room" + str(i), i, "addr" + str(i), randint(0, i), randint(0, i), [], {}, [], [])) for i in range(10)}
```

Comment:
Trochę zbyt skomplikowana linijka 😄. Tymczasowo zahardkodujemy listę konkretnych serwerów. Nie będziemy ich raczej losować 😃

Reply:
ja wiem, to takie na szybko było, nie chciało mi się za dużo pisać, ale chciałam sprawdzić, czy działa

Reply:
i nie wiem, czy zwracając listę roomów trzeba zwracać wszystkie pola klasy Room jak karty czy gracze (zapewne nie trzeba), ale tak jak pisałam - to dla testu :)

Reply:
Nieeee, zwracamy tylko to co potrzebuje front. Było mówione 😛 
![image](https://user-images.githubusercontent.com/19170699/81714960-f1fb0380-9477-11ea-8a14-984b0be66658.png)

Reply:
Aaa dobra, w treści taska jest podane, że ma być "adres do dołączenia". Okej, task jest oczywiście dość stary i niezaktualizowany stąd ta rozbieżność pewnie

Reply:
pola klasy Room mają się nazywać tak, jak w mockupie czy nie trzeba?

Reply:
Robisz to teraz? Bo właśnie integruję to API z obecnym masterem i klasą `Room` :P

Reply:
tak
ehhh, jakieś 5 minut temu to wypchnęłam :/